packages/run-protocol/test/amm/vpool-xyk-amm/test-liquidity.js

```diff
@@ -145,6 +145,33 @@ const makeAssertPayouts = (
   };
 };
 
+test('amm add issuers in parallel', async t => {
+  // Set up central token
+  const centralR = makeIssuerKit('central');
+  const moolaR = makeIssuerKit('moola');
+
+  const { amm } = await setupAmmServices(
+    t,
+    { committeeName: 'EnBancPanel', committeeSize: 3 },
+    centralR,
+  );
+
+  const [issuer1, issuer2, issuer3] = await Promise.all([
```

Comment:
This should also test adding the same issuer with the SAME keyword.

Reply:
good idea.